pkgs/development/tools/analysis/banana-vera/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, python310
+, tcl
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "banana-vera";
+  version = "1.3.0-python3.10";
+
+  src = fetchFromGitHub {
+    owner = "Epitech";
+    repo = "banana-vera";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-1nAKhUltQS1301JNrr0PQQrrf2W9Hj5gk1nbUhN4cXw=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [
+    python310
```

Comment:
Why is 310 hard coded here? Can we just use python3?

Reply:
I assume it's because upstream explicitly specified Python 3.10 [in the release](https://github.com/Epitech/banana-vera/releases/tag/v1.3.0-python3.10) and [in commit name](https://github.com/Epitech/banana-vera/commit/3a7d18d6249cd1790cc63fb9018e0914462ec219)

Reply:
Yes, the python version is pinned as python3.10